mira:    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    # Position Controllers ---------------------------------------
    roll_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: shoulder_pan_joint
      pid: {p: 1.0, i: 1.0, d: 0.0}
    # Add more here as needed
